website/docs/docs/cloud-integrations/configure-auto-exposures.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Auto-exposures"
+id: "configure-auto-exposures"
+sidebar_label: "Configure auto-exposures"
+description: "Import and auto-generate exposures from dashboards and understand how models are used in downstream tools for a richer lineage."
+image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
+---
+
+# Configure auto-exposures <Lifecycle status='beta' />
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. [Auto-exposures](/docs/collaborate/auto-exposures) integrates natively with Tableau (Power BI coming soon) and [auto-generates downstream lineage](/docs/collaborate/auto-exposures#view-auto-exposures-in-dbt-explorer) in dbt Explorer for a richer experience.
+
+Auto-exposures help data teams optimize their efficiency and ensure data quality by:
+
+- Helping users understand how their models are used in downstream analytics tools to inform investments and reduce incidents — ultimately building trust and confidence in data products.
+- Importing and auto-generating exposures based on Tableau dashboards, with user-defined curation.
+- Enabling the active exposure work to run models based on when exposures are updated or need to be updated, improving timeliness and reducing costs.
+
+## Prerequisites
+
+To access the features, you should meet the following:
+
+1. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+2. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+3. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
+4. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
```

Comment:
We may need an update for Tableau server based on what we find out from customers about Whitelisting the IP address (just an additional callout of the extra step for Server). I'll know more on Monday. Is it fine to make the update post it going live if needed?

Reply:
Yes, we can totally update after it goes live!